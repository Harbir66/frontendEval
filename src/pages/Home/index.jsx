import React from 'react';
import { Header, Footer, MainContainer } from '../../components';
import './Home.css';

function Home() {
  return (
    <div className="home">
      <Header />
      <MainContainer />
      <Footer />
    </div>
  );
}

export default Home;
