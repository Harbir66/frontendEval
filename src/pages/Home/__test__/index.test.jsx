import React from 'react';
import { render } from '@testing-library/react';
import Home from '../index';

jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn(),
}));

describe('Error Page', () => {
  it('should render', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
