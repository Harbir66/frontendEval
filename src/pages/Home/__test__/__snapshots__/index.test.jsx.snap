// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page should render 1`] = `
<DocumentFragment>
  <div
    class="home"
  >
    <header
      class="padding"
    >
      EVENTIFY
    </header>
    <div
      class="main-container padding"
    >
      <div>
        Loading...
      </div>
    </div>
    <footer
      class="padding"
    >
      <p>
        THEMES
      </p>
      <div
        class="themes"
      />
    </footer>
  </div>
</DocumentFragment>
`;
